### PR TITLE
Show the exception message if one exists

### PR DESCRIPTION
**Pull request by bomsy**
Associated Issue: #2677

### Summary of Changes

* Add exceptions message

### Screenshots
screen shot 2017-04-19 at 1 13 49 pm

### CODE REVIEW

File: src/components/SecondaryPanes/WhyPaused.js [hunk 1]
```diff
@@ -17,11 +17,20 @@ class WhyPaused extends Component {
     }
 
     const message = pauseInfo.getIn(["why"]).get("message");
-    if (!message) {
-      return null;
+    if (message) {
+      return dom.div({ className: "message" }, message);
+    }
+
+    const exception = pauseInfo.getIn(["why"]).get("exception").toJS();
```
Comment (clarkbw) [inline src/components/SecondaryPanes/WhyPaused.js]: I think you should be able to do `pauseInfo.getIn(["why", "exception"])` here.  I'm pretty sure I did the `getIn().get()` pattern above for `message` and I think I made a mistake.

**Comment by jasonLaster** [comment]
Looks like we got a test failure, which can be run in the WTR

https://github.com/devtools-html/debugger.html/blob/master/src/test/integration/runner.js#L78-L84

**Comment by bomsy** [comment]
hmmm ... tests seem to pass locally for me

**Comment by jasonLaster** [comment]
Hmm. I'll look

**Comment by jasonLaster** [comment]
Looks like we were catching an edge case where exceptions were strings!

* updated the code
* added another unit test

screen shot 2017-04-21 at 4 49 39 pm

**Comment by bomsy** [comment]
ahaa
